src/components/Homepage/BodySection/BodySection.tsx

```diff
@@ -39,6 +41,10 @@ export const BodySection = ({ section, images }: { section: SectionProps; images
   const singleColumn = columns == 1;
   const bottomMargin = sectionBottomMarginVariants[section.bottomMargin];
 
+  if (section.callToAction && checkLinkIsInternal(section.callToAction.href)) {
```

Comment:
wow this is bizarre

Reply:
we have to check like this, otherwise the `/docs` links won't work locally, and but will in production or they work locally, and break in production, this look up is used whenever the `<Link>` component is used